code/modules/mob/dead/observer/observer.dm

```diff
@@ -206,12 +206,25 @@ This is the proc mobs get to turn into a ghost. Forked from ghostize due to comp
 
 // This is the ghost's follow verb with an argument
 /mob/dead/observer/proc/ManualFollow(atom/movable/target)
-	if(target && target != src)
-		if(orbiting && orbiting == target)
-			return
+	if (!istype(target))
+		return
+
+	var/icon/I = icon(target.icon,target.icon_state,target.dir)
+
+	var/orbitsize = (I.Width()+I.Height())/2

```

Comment:
replace ```I.Width()``` with ```_dm_icon_size(I,1)``` and ```I.Height()``` with ```_dm_icon_size(I,2)```
since those procs literally do nothing but call those procs anyway, it's overhead.

and the ```/2``` should be ```*0.5```